GW2EIEvtcParser/EIData/Buffs/CommonBuffs.cs

```diff
@@ -163,12 +163,29 @@ internal static class CommonBuffs
             new Buff("Gourd Vibrations", GourdVibrations, Source.Gear, BuffClassification.Gear, BuffImages.OrbOfAscension),
             new Buff("Cat's Shadow", CatsShadow, Source.Gear, BuffClassification.Gear, BuffImages.OrbOfAscension),
             // Relics
-            new Buff("Relic of the Dragonhunter", RelicOfTheDragonhunter, Source.Gear, BuffStackType.StackingSomething, 999, BuffClassification.Gear, BuffImages.RelicOfTheDragonhunter), // Applied on target
+            new Buff("Relic Player Buff (Dragonhunter / Isgarren / Peitha)", RelicTargetPlayerBuff, Source.Gear, BuffClassification.Gear, BuffImages.Unknown),
```

Comment:
Should be BuffClassification.Hidden, does not serve any visible purpose to the end user.